### PR TITLE
Team player crud

### PR DESCRIPTION
**Pull request by tschaffer1618**
- Javascript search bar for player addition on team show page
- Link player names to player show page
- Test team_player crud functionality

### CODE REVIEW

File: app/controllers/user/team_players_controller.rb [hunk 1]
```diff
@@ -11,6 +11,23 @@ def destroy
...
+        flash[:success] = "Player added!"
+      else
+        flash[:error] = "Player already on team."
+      end
+    else
+      flash[:error] = "No player found matching that name."
+    end
+    redirect_to user_team_path(team)
+  end
+
```
Comment (fentontaylor) [inline app/controllers/user/team_players_controller.rb]: These should probably be private methods if we keep this logic here

File: app/controllers/user/team_players_controller.rb [hunk 2]
```diff
@@ -11,6 +11,23 @@ def destroy
...
+  def create_from_player_search
```
Comment (alect47) [inline app/controllers/user/team_players_controller.rb]: Thanks for re-naming this

Comment (tschaffer1618) [reply app/controllers/user/team_players_controller.rb]: The plan is to get these 2 merged into one create method, but I just renamed for now so we can all work out of the same code base.

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -57,7 +57,7 @@
     updated_user = User.first
 
     expect(updated_user.provider).to eq("google")
-    expect(updated_user.uid).to eq("12345678910")
+    expect(updated_user.uid).to eq("104534628262753737066")
```
Comment (alect47) [inline spec/models/user_spec.rb]: nice

**Pull request merged**